Index: modules/search.module
===================================================================
RCS file: /cvs/drupal/drupal/modules/search.module,v
retrieving revision 1.38
diff -u -r1.38 search.module
--- modules/search.module	21 Apr 2003 14:55:03 -0000	1.38
+++ modules/search.module	22 Apr 2003 23:03:05 -0000
@@ -305,8 +305,8 @@
   variable_set("remove_short", $edit["remove_short"]);
 }
 
-function search_view() {
-  global $edit, $type, $keys;
+function search_view($keys = NULL) {
+  global $edit, $type;
 
   if (user_access("search content")) {
     // Construct the search form:
@@ -357,7 +357,7 @@
 }
 
 function search_page() {
-
+  global $keys;
 
   switch (arg(1)) {
     case "help":
@@ -366,7 +366,7 @@
       theme("footer");
       break;
     default:
-      search_view();
+      search_view($keys);
   }
 }
 
